components/alert/alert.css

```diff
@@ -0,0 +1,110 @@
+@import url('/css/style.css');
+
+body {
+  background-image: none;
+}
+.wrapper .bottom-nav {
+  width: 100%;
+  background: #f9f9f9;
```

Comment:
declare variables and use here. Don't hard-code .

Reply:
Thank you Vikrant for your vaulable feedback